<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>590</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textOutput">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>590</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="simulationTimeOutput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Simulation Step: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator0Label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator1Label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="ElevatorLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elevators</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="elevator0Output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="elevator1Output">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="passenger0_starting_floor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="passenger1_starting_floor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="passenger3_starting_floor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="passenger2_starting_floor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="passenger0_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="passenger1_labell">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="passenger2_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="passenger3_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Starting_Floor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Starting Floor</string>
    </property>
   </widget>
   <widget class="QComboBox" name="passenger0_dest_floor">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="passenger2_dest_floor">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="passenger3_dest_floor">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Destination_Floor">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination Floor</string>
    </property>
   </widget>
   <widget class="QComboBox" name="passenger1_dest_floor">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="set_passenger_values_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="passenger0_button_press_step">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="passenger1_button_press_step">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="passenger2_button_press_step">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>190</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="passenger3_button_press_step">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>220</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Button_Press_Step">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Button Press Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="PassengerLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Passengers</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger0Output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger1Output">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger2Output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger3Output">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Fire_Alarm_Button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire Alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Help_Alarm_Button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Help Alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Door_Alarm_Button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>440</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Overload_Alarm_Button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Overload Alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Power_Out_Button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Out Alarm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElevator_Simulator">
    <property name="title">
     <string>Elevator Simulator</string>
    </property>
   </widget>
   <addaction name="menuElevator_Simulator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
